.gitignore

```diff
@@ -352,3 +352,6 @@ MigrationBackup/
 src/Playwright.sln.DotSettings
 src/Playwright/Drivers
 src/Playwright/playwright
+src/Playwright.Tests/Screenshots/CHROMIUM/test.png
+src/Playwright.Tests/Screenshots/FIREFOX/test.png
+src/Playwright.Tests/Screenshots/WEBKIT/test.png
```

Comment:
I think we could also just say `src/Playwright.Tests/Screenshots/**/*.png` to be a bit more generic here?

Reply:
Done!
I'm leaving the `test.png` because it's the only one we need to ignore there.

Reply:
Yeah, that's fine. I'm just thinking of how we can minimize changing this in the future :)